.github/workflows/main.yml

```diff
@@ -0,0 +1,42 @@
+name: CI
+on: [push]
+jobs:
+  build:
+    runs-on: ubuntu-latest
```

Comment:
Nice, so the CI runs on github actions now rather than CircleCI as our other libs? Probably simplifies things :+1:

Reply:
Yes I just used https://tsdx.io/ to scaffold the project and it came with this setup so that seemed like a good default to keep!